Makefile

```diff
@@ -45,13 +52,26 @@ venvs/ot2: ot2monorepoClone
 	popd && \
 	deactivate
 
-# OVERRIDE_SETTINGS_DIR must be set to use opentrons v3
-# (otherwise it will try to access /data dir during 'import opentrons')
+.PHONY: parse-ot1
+parse-ot1: venvs/ot1 $(OT1_OUTPUT_FILES)
+
+# Parse all OT1 python files
+$(BUILD_DIR)/%.ot1.py.json: protocols/%.ot1.py
+	mkdir -p $(dir $@) && \
+	source venvs/ot1/bin/activate && \
```

Comment:
This `&& \` is _not_ redundant because `make` executes each line in a separate shell invocation, so without the chaining the next line would not have the venv activated